pkg/varlinkapi/images.go

```diff
@@ -541,7 +541,7 @@ func (i *LibpodAPI) Commit(call iopodman.VarlinkCall, name, imageName string, ch
 			c <- err
 		}
 		c <- nil
-		close(c)
+		defer close(c)
```

Comment:
What is the difference here?

Reply:
the channel was closing early and causing panics